src/cmd/manage_nsfs.js

```diff
@@ -597,6 +597,10 @@ async function list_account_config_files(wide, show_secrets, filters = {}) {
     const options = {
         show_secrets: show_secrets || should_filter,
         decrypt_secret_key: show_secrets,
+        // in case we have an error on secret_key decryption 
+        // we will return the encrypted_secret_key
```

Comment:
I wouldn't return the encrypted secret key, why do you want to return it to the user?

Reply:
I thought it was another indication for the user that we had a decryption issue.
I changed it to remove this property, thanks.